benchmark/Microsoft.IdentityModel.Benchmarks/identitymodel.benchmarks.yml

```diff
@@ -57,3 +57,8 @@ scenarios:
       variables:
         filterArg: "*ValidateTokenAsyncTests*"
 
+  Base64UrlEncoderTests:
+    application:
+      job: benchmarks
+      variables:
+        filterArg: "*Base64UrlEncoderTests*"
```

Comment:
Is there a benefit on running these benchmarks in builds? I think these are good to run right now to verify the work. Any perf regressions to encoding/decoding should be caught by other validation benchmarks.

Reply:
I don't think they are today, correct? If not, let's open an issue figure out prioritization.

Reply:
No, these are new tests.

Reply:
Yes, but no benchmarks are run today as part of the PR builds, correct?

Reply:
Yes, I think so.